egs/librispeech/ASR/conformer_ctc/decode.py

```diff
@@ -56,6 +57,15 @@ def get_parser():
         "consecutive checkpoints before the checkpoint specified by "
         "'--epoch'. ",
     )
+
+    parser.add_argument(
+        "--scale",
```

Comment:
If this scale is only used for the nbest-oracle mode, perhaps that should be clarified, e.g. via the name and the documentation?  Right now it's a bit unclear whether this would affect other things

Reply:
I think it is also useful for other n-best rescoring methods, e.g., attention-decoder rescoring. Tuning this value can
change the number of unique paths in an n-best list, which can potentially affect the final WER.

I'm adding more documentation to clarify its usage.